manticore/platforms/linux.py

```diff
@@ -894,6 +922,14 @@ def load(self, filename):
                 elf_brk = k
 
         elf_entry = elf.header.e_entry
+        if function is not None:
+            symbolPC = self._find_symbol(function)
+            if symbolPC is None:
+                logger.error("No symbol for %s in %s", function, program)
+                #TODO raise exception
```

Comment:
Actually raise here?

Reply:
a normal Exception ? or is there a better subclass ?